### ISSUE TITLE
Retro-Bit Super Retro-Cade

### ISSUE THREAD

**Issue by kjetil-f** [farmerbb/RED-Project#85]
Using a USB to UART adapter with male to female jumper wire you can dump all of the included games. Even though you can, there is **no soldering required** to do this.

Use TeraTerm (PuTTY or other terminal emulators probably work as well), Set the port to COM5, Baud rate/speed to 115200 and language to UTF-8.

Backup the games to a FAT formatted SD card using the following command: cp -r /system/media/GAME/ /storage/extsd/

Based on this tutorial: http://blog.livedoor.jp/scrap_a/archives/22692919.html

**Comment by RealRelativeEase** [comment]
Thank you for the detailed instructions! Does this work on a European system as well?

**Comment by kjetil-f** [comment]
It does. On the US version it is possible to do with Lakka installed on an SD-card, but it seems like they have removed some access permissions on the EU version.

For the US version, check out [this thread on GBAtemp](https://gbatemp.net/threads/super-retro-cade.519483/). It explains the SD card method

**Comment by kjetil-f** [comment]
Here's a picture: https://global.discourse-cdn.com/standard10/uploads/retrogameboards/original/3X/8/8/8803090d1b33825c795f31f906e030256e722a42.jpeg

**Comment by RealRelativeEase** [comment]
I've got a USB TTL board and some female to male jumper wires, but I still can't connect to my device. I've got the wires connected (Ground to Ground, Receive to Transmit and vice versa), but I still can't access the device via TeraTerm or Putty. There's no additional power source required, other than the USB port, correct? Also, the lights on my USB TTL board will only flash when partially plugged in and turn off once it's fully plugged in.

**Comment by kjetil-f** [comment]
Yeah, you need to plug in the power as well. Don't know if the order matters, but what I did is that I first plugged in the jumper wires, pressed the power button in, started the software AND THEN plugged in the power cable to the console (cable already plugged to the power outlet of the wall).

FYI the markings on my adapter seems to have been marked incorrect so I connected it RX to RX and TX to TX. I found out when I read the reviews from Amazon.

**Comment by RealRelativeEase** [comment]
Thank you so much, it worked! Finally, I get to use these ROMs with better emulators.
